src/css/controls/imports/nextup.less

```diff
@@ -163,3 +163,7 @@
         visibility: visible;
     }
 }
+
+.jw-nextup-firefox-pip-fix {
+    background: rgba(38, 38, 38, 1);
```

Comment:
Wouldn't it be better to use a method like [fade](https://lesscss.org/functions/#color-operations-fade) to only set the opacity and use the `@overlay-background-color` variable instead of using a static value?
I'm not sure if this would affect or override custom themes but that might also be a risk

Reply:
I can change it to
`.jw-nextup-firefox-pip-fix {
    background: fade(@overlay-background-color, 100%);
}`
and it seems to work the same for firefox and chrome